src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -15,107 +30,216 @@ type PipelineRunDetailsTabProps = {
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Namespace</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun?.metadata.namespace}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Labels</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {getMetadataList(pipelineRun?.metadata?.labels)}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Annotations</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {getMetadataList(pipelineRun?.metadata?.annotations)}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Created at</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <Timestamp timestamp={pipelineRun?.metadata.creationTimestamp} />
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Duration</DescriptionListTerm>
+                <DescriptionListDescription>{duration}</DescriptionListDescription>
+              </DescriptionListGroup>
+            </DescriptionList>
+          </FlexItem>
+        </Flex>
+
+        <Flex flex={{ default: 'flex_3' }}>
+          <FlexItem>
+            <DescriptionList
+              data-test="pipelinerun-details"
+              columnModifier={{
+                default: '1Col',
+              }}
+            >
+              <DescriptionListGroup>
+                <DescriptionListTerm>Status</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {getBuildStatusIcon(status)} {status}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Message</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {pipelineRun?.status?.conditions[0].message}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>Log snippet</DescriptionListTerm>
+                <DescriptionListDescription>
+                  <CodeBlock>
+                    <CodeBlockCode id="code-content">
+                      {
+                        pipelineRun?.status?.taskRuns[`${pipelineRun?.metadata.name}-show-summary`]
+                          .status.taskSpec.steps[0].script
+                      }
+                    </CodeBlockCode>
```

Comment:
`show-summary` is a finally task. Not sure if we can rely on this task always being present. Also there won't be a result unless the finally task is run which means it won't be present while the pipeline is currently running.

We may want to create a story to bring in a log snippet similar to that of the dev console.

Reply:
i'll leave this as-is for now, and add a separate story and link it to the original JIRA